site/_components/item-grid.njk

```diff
@@ -10,14 +10,15 @@
     @param {number} modifiers.shapeOn: Index of the nth item of the grid that will have a shape in the background.
         A value equals to 0 will turn off the shape. Defaults to 0.
     @param {string} modifiers.variant: If 'icon', it will render the icon-card. if 'subnav' it will render card-subnav. if 'community' it will render external-community.
+    @param {string} modifiers.heading (optional): Designate the section heading typography class.
 
     @param {string} locale: Code for the current locale, used when 'modifiers.variant' is 'subnav'
 
     @param {string} defaultText: Default text used when 'modifiers.variant' is 'icon'.
  #}-->
 {% from 'external-community.njk' import externalCommunity %} {% from 'card-subnav.njk' import cardSubnav %} {% from 'icon-card.njk' import iconCard %} {% from 'card.njk' import card %} {% macro itemGrid(title, items, modifiers, locale, defaultText) %} {% if items.length > 0 %}
 <section class="item-grid {{modifiers.custom}} {{'item-grid--icon-card' if modifiers.variant == 'icon'}} {{ 'item-grid--subnav' if modifiers.variant == 'subnav' }} {{'item-grid--community' if modifiers.variant == 'community'}} {{'item-grid--circled' if modifiers.shapeOn }}">
-  <h3 class="item-grid__title type--h3">{{title}}</h3>
-  <div class="item-grid__content">{% for item in items %} {% if modifiers.variant == 'icon' %} {{iconCard(item, defaultText)}} {% elif modifiers.variant == 'subnav' %} {% if item.icon and item.desc %} {{cardSubnav(item, locale)}} {% endif %} {% elif modifiers.variant == 'community' %} {{externalCommunity(item.logo, item.cta)}} {% else %} {{ card(item) }} {% endif %} {% endfor %}</div>
+  <h3 class="item-grid__title {% if modifiers.heading %} type--{{modifiers.heading}} {% else %} type--h1 {% endif %}">{{title}}</h3>
```

Comment:
Out of curiosity, why aren't we preserving the existing default (`h3`) if this new optional modifier isn't included?

Reply:
`type--h1` is the most used treatment for this headline within homepage designs, so it seems a reasonable default. Would it make more sense to change the tag to an h1 and have no `else` for the optional modifier?

Reply:
I think it's OK as is. Let's revisit once everything's merged whether we need to change the actual tag too